src/main/java/games/strategy/triplea/oddscalc/OddsCalculatorPanel.java

```diff
@@ -352,16 +319,14 @@ private void updateStats() {
     }
     final AtomicReference<AggregateResults> results = new AtomicReference<>();
     final WaitDialog dialog =
-        new WaitDialog(this, "Calculating Odds (" + calculator.getThreadCount() + " threads)", new AbstractAction() {
+        new WaitDialog(this, "Calculating Odds", new AbstractAction() {
```

Comment:
Note to myself: When modifying the WaitDialog Constructor to accept an ActionListener instead of an Action, this abstract action can be replaced with a lambda expression
Actually, the only public usage of the second constructor is this line, so maybe within the scope of this PR, I'll let you decide

Nit: This line now can now bemerged with the line above

Reply:
The update here was to change the message, I want to keep the diff and any potential rollbacks to exactly that.

Reply:
Sorry, I don't quite follow on the line merge.
If this were refactored, I would suggest merging wait panel and dialog with JDialogBuilder so we could write something like:
```
JDialogBuilder.builder()
  .title("CalculatingOdds")
  .blocking(true)
  .cancelAction(() -> cancelAction())
  .showDialog();
```

Reply:
Line merge just meant this line now fits in the same line as the declaration.

Your suggestion would work too as well, but would require far more work, and is clearly out-of-scope, while changing a param type to an interface type is far easier to do